gui/src/containers/JobSpecRuns.js

```diff
@@ -73,26 +122,36 @@ export class JobSpecRuns extends Component {
   }
 }
 
-const renderLatestRuns = ({jobSpecId, classes, latestJobRuns, jobRunsCount, pageSize}, state, handleChangePage) => (
-  <Card>
-    <JobRunsList jobSpecId={jobSpecId} runs={latestJobRuns} />
-    <TablePagination
-      component='div'
+const renderLatestRuns = (props, state, handleChangePage) => {
+  const { jobSpecId, latestJobRuns, jobRunsCount, pageSize } = props
+  const TableButtonsWithProps = () => (
+    <TableButtons
+      {...props}
       count={jobRunsCount}
-      rowsPerPage={pageSize}
-      rowsPerPageOptions={[pageSize]}
-      page={state.page}
-      backIconButtonProps={{'aria-label': 'Previous Page'}}
-      nextIconButtonProps={{'aria-label': 'Next Page'}}
       onChangePage={handleChangePage}
-      onChangeRowsPerPage={() => {} /* handler required by component, so make it a no-op */}
+      page={state.page + 1}
+      specID={jobSpecId}
+      rowsPerPage={pageSize}
     />
-  </Card>
-)
+  )
+  return (
+    <Card>
+      <JobRunsList jobSpecId={jobSpecId} runs={latestJobRuns} />
+      <TablePagination
+        component='div'
+        count={jobRunsCount}
+        rowsPerPage={pageSize}
+        rowsPerPageOptions={[pageSize]}
+        page={state.page}
```

Comment:
Like you said in https://github.com/smartcontractkit/chainlink/pull/457 this is 0 based.

![image](https://user-images.githubusercontent.com/635121/43150329-a3247b34-8f37-11e8-8da5-710da46ab3ea.png)

Perhaps we can just `state.page-1` here and leave everything else 1-based. I'll leave it up to you, but as it stands now I think we're juggling between the two a lot.